test/specs/column/tests/02.reference_column.js

```diff
@@ -749,16 +749,16 @@ exports.execute = function (options) {
 
                     it ("otherwise, if asset column has filenameColumn and it's value is not empty, should use it for caption.", function () {
                         val = assetRefCompactCols[10].formatPresentation({"col_asset_3": "https://example.com", "col_filename": "filename"}).value;
-                        expect(val).toEqual('<a href="https://example.com?uinit=1&amp;cid=test" download="" class="download deriva-url-validate">filename</a>', "value missmatch.");
+                        expect(val).toEqual('<a href="https://example.com?uinit=1&amp;cid=test" download="" class="download asset-permission">filename</a>', "value missmatch.");
```

Comment:
`asset-permission` shouldn't be in the expected value, right? (it's not the same origin).

Reply:
I think the issue here is the input: `https://example.com`. It doesn't have any `/`. I don't think this is really a valid input so you should just change it to `https://example.com/file.png` or something that is valid.

Also you could change all these links to be relative, and then add a special `it` just to test the same-origin logic that you've added. When I wrote these test cases the input didn't matter that much, so you might want to modify them.

Reply:
Ah yes, I didn't even realize that was happening here.